### DISCUSSION TITLE
Workshop November 24th, 2021

### DISCUSSION THREAD

**Discussion by holyjak** [holyjak/interactive-dev-wshop#2] [General]
A place to share your questions, ideas, and feedback regarding the Clojure/Oslo workshop on November 24th, 2021.

**Comment by gam** [comment]
For step 8 in the intro exercises, I found using `partial` easier than creating an anonymous function (with `fn`).
YMMV, and I guess it presupposes knowledge of currying/partial function application.

**Reply by Zeniten** [reply]
;; 7. Define a vector of numbers 40 - 45 called nums (just type them out 😀)

I first didn't read that I should call the vector nums. I first called it numbers. I evaluated and moved on. Then I noticed that the text demanded (😀) that I called it nums. I renamed it and started task 8. After having written an expression, I got an exception when trying to evaluate it. The problem was that I had not evaluated the new def named nums. Somebody else experienced the same problem during the workshop, though I'm not sure if it they followed a similar process, so maybe this task should be reworded?

**Reply by Zeniten** [reply]
Perhaps something like?:
;; 7. Define a vector called nums containing the numbers 40 - 45 ...

**Reply by holyjak** [reply]
@gam Yes but that would require teaching one more function (and concept). The need here is to find _minimal_ necessary subset of Clojure :-)

**Reply by holyjak** [reply]
> Perhaps something like?: ;; 7. Define a vector called nums containing the numbers 40 - 45 ...

Thank you, fixed! https://github.com/holyjak/interactive-dev-wshop/blob/master/src/interactive/server.clj#L66

**Reply by gam** [reply]
> The need here is to find _minimal_ necessary subset of Clojure :-)

I know, and completely agree with the choice. 
My comment was more of an observation, in retrospect. :)
